### DISCUSSION TITLE
Autenticação QFieldCloud

### DISCUSSION THREAD

**Discussion by yamaciro** [opengisch/QField#2517] [Q&A]
Boa Tarde,

Instalei a versão 4.0.0-BETA18 do QFieldSync no Qgis 3.16.16-Hannover. Finalizei o cadastro no site http://app.qfield.cloud e criei meu login de acesso, mas não estou conseguindo autenticar o QFieldCloud no Qgis.

At, 

I installed version 4.0.0-BETA18 of QFieldSync on Qgis 3.16.16-Hannover. I finished the registration at http://app.qfield.cloud and created my login, but I am not able to authenticate QFieldCloud in Qgis.

**Comment by suricactus** [comment]
@yamaciro thanks for reporting your issue. The problem has been acknowledged and fixed with QFIeldSync BETA 19.
